client/extensions/woocommerce/app/order/order-create/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionHeader from 'woocommerce/components/action-header';
+import Button from 'components/button';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import Main from 'components/main';
+
+class OrderCreate extends Component {
+
+	render() {
+		const { className, site, translate } = this.props;
+		const breadcrumbs = [
+			( <a href={ getLink( '/store/orders/:site/', site ) }>{ translate( 'Orders' ) }</a> ),
+			( <span>{ translate( 'New Order' ) }</span> ),
+		];
+		return (
+			<Main className={ className }>
+				<ActionHeader breadcrumbs={ breadcrumbs }>
+					<Button primary>{ translate( 'Save Order' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Save New Order' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-create-order-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).